.gitignore

```diff
@@ -0,0 +1 @@
+credentials.json
```

Comment:
Will create a separate file for each one's to store their individual key, so it doesn't push for the public repo